packages/zowe-explorer-api/src/security/KeytarCredentialManager.ts

```diff
@@ -131,7 +169,7 @@ export class KeytarCredentialManager extends AbstractCredentialManager {
         // Throw an error if credentials could not be found
         if (password == null && !optional) {
             throw new ImperativeError({
-                msg: localize("errorHandling.loadCredentials", "Unable to load credentials."),
+                msg: "Unable to load credentials.",
```

Comment:
localization will be removed?

Reply:
We could look at adding it back since it's out of the profiles folder and make sure it doesn't break anything.  Let me test it locally.  I took it out due to comments made when it was in the profiles folder.

Reply:
I would like to work on some unit tests for the APIs when I get time, especially a check for imports of nls and vscode in the profiles folder so we don't run into this issue again.

Reply:
ok. no worries 👍 I'll also run the tests 😊